### DISCUSSION TITLE
Add more layout adjustability to Node SpinBox

### DISCUSSION THREAD

**Discussion by DannyDevSe** [godotengine/godot-proposals#8644] [GUI]
![Untitled(1)](https://github.com/godotengine/godot-proposals/assets/102230735/8ce1cd23-5de6-47f8-bb6a-60b3951aa18e)

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot-proposals/issues/5139.

Could you describe some use cases for making this adjustment possible?

Note that the "up" and "down" icons are currently part of a single icon, so they can't be drawn on opposite sides unless we use different icons in this case.
